internal/handler/handler.go

```diff
@@ -0,0 +1,91 @@
+package handler
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/ffddorf/unms-exporter/exporter"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"github.com/sirupsen/logrus"
+)
+
+type Handler struct {
+	base    *prometheus.Registry
+	targets map[string]*exporter.Exporter
+	log     logrus.FieldLogger
+}
+
+func New(logger logrus.FieldLogger, targets map[string]string) (http.Handler, error) {
```

Comment:
this doesn't look like it ever returns an error. let's remove the second return value

Reply:
Done.